dev-requirements.txt

```diff
@@ -0,0 +1,9 @@
+mock==1.0.1
+pytest-cov==2.5.1
+coverage==3.7.1
+boto3>=1.7.28
+#google-cloud==0.33.1
+google-cloud-storage==1.6.0
+-e git+https://git@github.com/uc-cdis/cdislogging.git@0.0.2#egg=cdislogging
+-e git+https://git@github.com/uc-cdis/indexclient.git@1.5.6#egg=indexclient
```

Comment:
Why are we also including `indexclient-1.5.2.zip` as a zip file? this seems like it'll be difficult to maintain. also, the dev-requirements have a different version `1.5.6`

Reply:
We run on remote machines so we have to pack all dependencies. 
Indeed it is 1.5.6 but the there is error in setup.py of indexclient that makes the build as 1.5.2